FirebaseStorage/Tests/SwiftIntegration/StorageIntegration.swift

```diff
@@ -357,6 +409,33 @@ class StorageIntegration: XCTestCase {
     waitForExpectations()
   }
 
+  func testUnauthenticatedSimpleGetFileWithCompletion() throws {
+    let expectation = self.expectation(description: #function)
+    let ref = storage.reference(withPath: "ios/public/cookie")
+    let tmpDirURL = URL(fileURLWithPath: NSTemporaryDirectory())
+    let fileURL = tmpDirURL.appendingPathComponent("cookie.txt")
+    let data = try XCTUnwrap("Here's a 🍪, yay!".data(using: .utf8), "Data construction failed")
+
+    ref.putData(data, metadata: nil, completion: { metadata, error in
+      XCTAssertNotNil(metadata, "Metadata should not be nil")
+      XCTAssertNil(error, "Error should be nil")
+
+      ref.write(toFile: fileURL) { url, error in
+        XCTAssertNil(error, "Error should be nil")
+        do {
+          let url = try XCTUnwrap(url, "Failed to unwrap url")
+          XCTAssertEqual(fileURL, url)
+          let stringData = try String(contentsOf: fileURL, encoding: .utf8)
+          XCTAssertEqual(stringData, "Here's a 🍪, yay!")
```

Comment:
Nit: it's probably good to have this in a single place - defined as a String above before we get the data from it. Then we can compare string data to the constant instead of having the value in two different places.